### ISSUE TITLE
Error after upgrading to storybook 7.1

### ISSUE THREAD

**Issue by drewbrend** [JesusTheHun/storybook-addon-react-router-v6#42]
I get this error when attempting to upgrade to storybook 7.1.1, my current version which works fine is v7.0.27.

> ../../node_modules/storybook-addon-react-router-v6/dist/index.d.ts(1,78): error TS2307: Cannot find module '@storybook/preview-api/dist/storybook-channel-mock-a19cd233' or its corresponding type declarations

Looks like this comes from here:

```typescript
import * as _storybook_preview_api_dist_storybook_channel_mock_a19cd233 from '@storybook/preview-api/dist/storybook-channel-mock-a19cd233';
import React from 'react';
import { RouteObject } from 'react-router';

declare const withRouter: _storybook_preview_api_dist_storybook_channel_mock_a19cd233.M;
```

Which I think is what this compiles to:

https://github.com/JesusTheHun/storybook-addon-react-router-v6/blob/c7063b8f74f31333310e373cc0ed5a9972344e43/src/features/decorator/withRouter.tsx#L7

I was able to reproduce this with both v2.0.0 and v1.0.2.

**Comment by JesusTheHun** [comment]
Hi @drewbrend, can you confirm `v2.0.1` fixes the issue ?

**Comment by JesusTheHun** [comment]
I confirmed the fix when I made the release `v2.0.1` so I'm closing this without your own validation. Feel free to reopen this if the issue persists.

**Comment by drewbrend** [comment]
Yup, just verified now and it's fixed.  Thanks!
